lib/util.js

```diff
@@ -161,9 +161,10 @@ function stylizeNoColor(str, styleType) {
 function arrayToHash(array) {
   var hash = Object.create(null);
 
-  array.forEach(function(val) {
+  for (var i = 0; i < array.length; i++) {
```

Comment:
I think that avoiding accessing the length property each time will speed up it a little more.
```js
var l = array.length;
var val;
var i;

for (i = 0;  i < l; i++) {
  val = array[i];
  hash[val] = true;
}
```